app/src/main/java/com/x8bit/bitwarden/data/auth/datasource/disk/AuthDiskSourceImpl.kt

```diff
@@ -55,6 +58,7 @@ class AuthDiskSourceImpl(
     encryptedSharedPreferences: SharedPreferences,
     sharedPreferences: SharedPreferences,
     legacySecureStorageMigrator: LegacySecureStorageMigrator,
+    private val vaultDiskSource: VaultDiskSource,
```

Comment:
The other disk sources shouldn't really interact with each other. They are meant to be completely isolated in that way. Can this be done at a level above?